scripts/prepare-smoke.sh

```diff
@@ -7,6 +7,7 @@ npm run webpack-bundle
 npm run pack:prepare
 
 cp ./redocly-cli.tgz ./openapi-core.tgz ./dist/bundle.js ./__tests__/smoke/
+cp ./redocly-cli.tgz ./openapi-core.tgz ./__tests__/smoke-plugins
```

Comment:
I'd move that to a separate script. It's being called twice anyway: https://github.com/Redocly/redocly-cli/pull/1749/files#diff-e5a87d0236e416c1fa636174d597bd7628a7434d6fe5bacbe4badd042b94804bR40

Reply:
Created a separate script for plugins smoke preparation